simulator/examples/offscreen.rs

```diff
@@ -1,3 +1,8 @@
+//! # Example: Off screen
```

Comment:
This is another example that could be removed IMO. It almost looks like a test and not an example.